HUD/GetMouseEvent.md

```diff
@@ -1,15 +1,15 @@
 ---
 ns: HUD
-aliases: ["0x632B2940C67F4EA9"]
+aliases: ["0x632B2940C67F4EA9", "_GET_SCALEFORM_MOVIE_CURSOR_SELECTION"]
 ---
-## _GET_SCALEFORM_MOVIE_CURSOR_SELECTION
+## GET_MOUSE_EVENT
  
 ```c
 // 0x632B2940C67F4EA9
-BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* context, int* slotIndex);
+BOOL GET_MOUSE_EVENT(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* context, int* slotIndex);
 ```
 
-Gets mouse selection data from scaleforms with mouse support. Must be checked every frame.
+Gets current mouse event details, triggered by a scaleform.
```

Comment:
This might be better off as "Gets the current mouse event details when triggered by a scaleform"

It might be a good idea to include the fact that it has to be ran every frame even if it might be implied.